frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart

```diff
@@ -175,19 +168,27 @@ class _CalendarPageState extends State<CalendarPage> {
     );
   }
 
-  Widget _buildCalendar(EventController eventController, int firstDayOfWeek) {
-    return Expanded(
-      child: Padding(
-        padding: GridSize.contentInsets,
-        child: MonthView(
+  Widget _buildCalendar(
+    BuildContext context,
+    EventController eventController,
+    int firstDayOfWeek,
+  ) {
+    final isInDocument = context.read<DocumentBloc?>() != null;
```

Comment:
I'm not sure if I like this. I think I'd prefer a field in the `CalendarPage` eg. `bool shrinkWrap` or `bool fixedHeight`.

Reply:
Good idea, how would you do this? I still need the build context so it has to be in the build method. Or do you mean passing in the bool in the constructor? I don't think that's a bad idea since other database views could also benefit from this, but I don't want to make the code for DatabaseViewWidget complicated just for the calendar.

Reply:
I see what you mean. Let me check what other options we have.

Reply:
@Xazin @richardshiue Any updates? If not, I would prefer a solution that involves adding a parameter called 'shrinkWrap'.

Reply:
I think it's fine to default `shrinkWrap` to false, and then in the relevant database block builder simply set it to true.

Reply:
Sounds good to me. @richardshiue What do you think of this solution?

Reply:
I'm a bit confused. Do you mean we pass shrinkwrap through constructors from database_view_block_component down to calendar_page? I wanna make sure before I commit

Reply:
Yes. Use shrinkWrap instead of the isInDocumrnt to identify whether the database view needs more space or not.